src/views/ItemView.js

```diff
@@ -430,7 +430,7 @@ class ItemView extends React.Component {
         {`${staffMember.personal.lastName}, ${staffMember.personal.firstName} ${staffMember.personal.middleName || ''}`}
       </Link> :
       '-';
-    const servicePointName = get(servicePoints, 'records[0].name', '-');
+    let servicePointName = get(servicePoints, 'records[0].name', '-');
```

Comment:
```suggestion
    const checkInDate = getDateWithTime(get(item, ['lastCheckIn', 'dateTime']));
    const servicePointName = checkInDate === '-' ? '-' : get(servicePoints, 'records[0].name', '-');
```

Reply:
@uladislausamets I agree, It looks better. I made the changes.